app/controllers/settings/general.js

```diff
@@ -33,7 +33,7 @@ export default Controller.extend(SettingsSaveMixin, {
     }),
 
     iconImageSource: computed('model.icon', function () {
-        return this.get('model.icon') || '';
+        return this.get('model.icon') || '/favicon.ico';
```

Comment:
So I haven't tested this - but since the icon location is just directly dumped into the img _src_ attribute, won't this cause issues with Ghost blogs in a subdirectory?

If I'm misunderstanding what the purpose of the default is then let me know 😄